docs/exercises/5-multi_region.rst

```diff
@@ -396,11 +401,11 @@ key for encryption/decryption.
 
             ./revoke_grant.sh
 
-You can now go back to the cloudwatch logs and see the application return to using the local key for encryption
-and decryption.
+You can now go back to the cloudwatch logs and see the application return to using the local key in us-east-2 for
```

Comment:
CloudWatch

Again suggest moving away from "local key" and instead pedantically say "CMK in $region"